BTCPayServer/wwwroot/js/wallet/wallet-camera-scanner.js

```diff
@@ -1,7 +1,7 @@
 window.addEventListener("load", () => {
     let $input = null;
     initCameraScanningApp("Scan address or payment link", data => {
-        if (data.includes('?') || $input == null) {
+        if (data.includes(':') || $input == null) {
```

Comment:
Let's change it to this to test for both options:

```suggestion
        if (data.match(/[:?]/) || $input == null) {
```

Reply:
Sure, but I'm curious why? There can never be "?" without a ":" in a [BIP21 URI](https://github.com/bitcoin/bips/blob/master/bip-0021.mediawiki)